footer.php

```diff
@@ -3,26 +3,26 @@
         <div class="grid-row u-align-center">
           <div class="grid-item item-s-8">
             <ul>
-            <li><a href="<?php echo home_url('about-us/'); ?>"><?php echo __('[:es]Acerca de[:en]About us'); ?></a></li>
-            <li><a href="<?php echo home_url('journal/'); ?>"><?php echo __('[:es]Journal[:en]Journal'); ?></a></a></li>
-            <li><a href="<?php echo home_url('recipes/'); ?>"><?php echo __('[:es]Recetas[:en]Recipes'); ?></a></a></li>
-            <li><a href="<?php echo home_url('catering/'); ?>"><?php echo __('[:es]Catering[:en]Catering'); ?></a></a></li>
-            <li><a href="<?php echo home_url('friends/'); ?>"><?php echo __('[:es]Amigos[:en]Friends'); ?></a></a></li>
-            <li><a href="<?php echo home_url('press/'); ?>"><?php echo __('[:es]Prensa[:en]Press'); ?></a></a></li>
+            <li><a href="<?php echo home_url('about-me/'); ?>"><?php echo __('[:es]Acerca de[:en]About me'); ?></a></li>
+            <li><a href="<?php echo home_url('journal/'); ?>"><?php echo __('[:es]Journal[:en]Journal'); ?></a></li>
+            <li><a href="<?php echo home_url('recipes/'); ?>"><?php echo __('[:es]Recetas[:en]Recipes'); ?></a></li>
+            <li><a href="<?php echo home_url('pichon/'); ?>"><?php echo __('[:es]Pichón[:en]Pichón'); ?></a></li>
+            <li><a href="<?php echo home_url('clients/'); ?>"><?php echo __('[:es]Clientes[:en]Clients'); ?></a></li>
+            <li><a href="<?php echo home_url('press/'); ?>"><?php echo __('[:es]Prensa[:en]Press'); ?></a></li>
             </ul>
           </div>
 
           <div class="grid-item item-s-8">
             <ul>
-              <li><a href="https://www.facebook.com/pichondf/" target="_blank">Facebook</a></li>
-              <li><a href="https://twitter.com/Pichoooooon" target="_blank">Instagram</a></li>
-              <li><a href="" target="_blank">Twitter</a></li>
+              <li><a href="https://instagram.com/nikinakazawa/" target="_blank">My Instagram</a></li>
+              <li><a href="https://instagram.com/pichondf/" target="_blank">Pichón Instagram</a></li>
+              <li><a href="https://instagram.com/netamezcal/" target="_blank">Neta Mezcal Instagram</a></li>
```

Comment:
Could this be better as a repeatable group with URL and Text fields? I feel like her affiliations will change maybe